during_the_workshop.md

```diff
@@ -203,11 +203,40 @@ If you are on Windows and you create a `run.sh` file yourself, instead of using
 
 ### 03: Run the containers
 
-Running each container with `docker run` is fairly straightforward. However, the containers will need a way to communicate; we suggest you mount a [**named volume**](https://docs.docker.com/storage/volumes/). Use the same volume for both containers so that the "webapp" container can access files created by the "cliapp" container.
+Try running each container with `docker run` and make sure they work.
+
+The cliapp should run its script and then exit.
+
+The webapp should serve a website until it is stopped. You will also need to `--publish` a port when running the web app container in order to view the website at `http://localhost:<published-port-number>/`.
+
+### 04: Mount a shared volume
+
+We ca now run our containers independently, but we want to display the data the cliapp generates in the webapp. For this containers will need a way to communicate; we suggest you mount a [**named volume**](https://docs.docker.com/storage/volumes/). Use the same volume for both containers so that the "webapp" container can access files created by the "cliapp" container.
```

Comment:
```suggestion
We can now run our containers independently, but we want to display the data the cliapp generates in the webapp. For this, containers will need a way to communicate; we suggest you mount a [**named volume**](https://docs.docker.com/storage/volumes/). Use the same volume for both containers so that the "webapp" container can access files created by the "cliapp" container.
```